network/core_test.go

```diff
@@ -246,7 +254,7 @@ func TestRandomTreeNetwork(t *testing.T) {
 		conns = append(conns, conn)
 	}
 	close(wait)
-	waitForRoot(conns, 10*time.Second)
+	waitForRoot(conns, 30*time.Second)
```

Comment:
would it make sense to make the `30*time.Seconds` into a constant since it is used multiple places or are they distinct cases that just happen to have the same value?

Reply:
Possibly... the tests could take different amounts of time to converge, so technically they all just happen to have the same amount of time, but 30s is meant to be "forever" as far as the tests are concerned.